corehq/apps/hqwebapp/tasks.py

```diff
@@ -15,6 +17,23 @@
 from corehq.util.log import send_HTML_email
 
 
+def mark_subevent_gateway_error(messaging_event_id, error, retrying=False):
```

Comment:
Should this check if `messaging_event_id` is `None` (emails that don't come from our messaging framework), either here or in the email task before calling this?

Reply:
I was thinking this would just hit the `DoesNotExist`, but now I think that's a bad idea to do an extra non-required DB hit. Will update.